app/helpers/application_helper/button/miq_ae_domain_priority_edit.rb

```diff
@@ -0,0 +1,10 @@
+class ApplicationHelper::Button::MiqAeDomainPriorityEdit < ApplicationHelper::Button::Basic
+  def disabled?
+    (User.current_tenant.visible_domains.length < 2)
+  end
+
+  def calculate_properties
+    super
+    self[:title] = N_("You need two or more domains to edit domains priorities") if disabled?
```

Comment:
The correct text here should read:

`You need two or more domains to edit domain priorities`

first is plural, the other is singular.